callBackPractice/callBackPractice.js

```diff
@@ -58,6 +77,13 @@ contains(foods, 'ribeye', (result) => {
 
 // Write a function called removeDuplicates that removes all duplicate values from the given array.
 // Pass the array to the callback function.  Do not mutate the original array.
+*/
+const removeDuplicates = (item, cb) => {
+  let unique = item.filter(function(elem, index, self) {
```

Comment:
Awesome use of filter here. Maybe I have you explain this to the other students later on? I think they'd all benefit from your code design!